src/main/java/reiff/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,53 @@
+package reiff.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.setColor(Color.BLACK);
+
+        double interval = 0.1;
+
+        double time = 0;
+        double prevX = 0;
+        double prevY = 0;
+
+        while (time <= projectile.getTotalTime()) {
+            double x = projectile.getX(time);
+            double y = projectile.getY(time);
+
+            g.drawLine((int) prevX, (int) -prevY, (int) x, (int) -y);
+
+            prevX = x;
+            prevY = y;
+
+            time += interval;
+        }
+
+        g.setColor(Color.BLUE);
+        double x = projectile.getX(projectile.getTotalTime() / 2);
+        double y = projectile.getPeakY();
+        g.fillOval((int) x, -(int) y, 10, 10);
```

Comment:
The oval doesn't get drawn with the center at x,-y.

The oval gets drawn with the upper left hand corner being at x,y and the width and height being 10.

If you want the center at x,y you need to modify the point.